### PR TITLE
add fabs jax implementation

### PR DESCRIPTION
**Pull request by glasalm**
Closes https://github.com/unifyai/ivy/issues/9704

### CODE REVIEW

**Comment by glasalm** [comment]
https://jax.readthedocs.io/en/latest/_autosummary/jax.numpy.fabs.html

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by glasalm** [comment]
@ZoeCD please review my PR, I am waiting for it a week already

**Comment by ZoeCD** [comment]
Hi! The test is not passing. Please check the details on `test-frontend-jax / run-nightly-tests (pull_request)` 🙂

**Comment by glasalm** [comment]
> Hi! The test is not passing. Please check the details on `test-frontend-jax / run-nightly-tests (pull_request)` 🙂

Hello @ZoeCD! The test is passing for me locally. Can you show me where the error is?

```
root@codespaces-d5ac88:/workspaces/ivy# pytest ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py::test_jax_numpy_fabs
Use Database in ReadOnly Mode with local caching !
=============================================================== test session starts ===============================================================
platform linux -- Python 3.8.10, pytest-7.1.2, pluggy-1.0.0
rootdir: /workspaces/ivy
plugins: json-report-1.5.0, metadata-2.0.4, hypothesis-6.55.0, typeguard-2.13.3
collected 4 items                                                                                                                                 

ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py ....                                                                      [100%]

================================================================ warnings summary =================================================================
../../usr/local/lib/python3.8/dist-packages/flatbuffers/compat.py:19
  /usr/local/lib/python3.8/dist-packages/flatbuffers/compat.py:19: DeprecationWarning: the imp module is deprecated in favour of importlib; see the module's documentation for alternative uses
    import imp

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
========================================================== 4 passed, 1 warning in 9.44s ===========================================================
```

**Comment by glasalm** [comment]
Unfortunately I cannot see the output on `test-frontend-jax / run-nightly-tests (pull_request)`

image

**Comment by glasalm** [comment]
@ZoeCD kindly waiting for your review :)

**Comment by ZoeCD** [comment]
I just noticed the error I saw is happening for other functions too. I'll take a deeper look tomorrow and let you know what I found

**Comment by glasalm** [comment]
@ZoeCD please let me know once you have time to check it out!

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 1]
```diff
@@ -1489,6 +1489,37 @@ def test_jax_numpy_fmin(
...
+@handle_frontend_test(
+    fn_tree="jax.numpy.fabs",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+    ),
+)
+def test_jax_numpy_fabs(
+    *,
+    dtype_and_x,
+    as_variable,
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:1502]: You don't need this. This is what's causing the error :)

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 2]
```diff
@@ -1489,6 +1489,37 @@ def test_jax_numpy_fmin(
     )
 
 
+# fabs
+@handle_frontend_test(
+    fn_tree="jax.numpy.fabs",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:1496]: You need to test for positive and negative numbers. Make sure the floats are signed

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py [hunk 3]
```diff
@@ -1489,6 +1489,37 @@ def test_jax_numpy_fmin(
...
+    num_positional_args,
+    native_array,
+    on_device,
+    fn_tree,
+    frontend,
+):
+    input_dtype, x = dtype_and_x
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        as_variable_flags=as_variable,
```
Comment (ZoeCD) [inline ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:1512]: You're adding extra arguments that are not really needed

**Review by ZoeCD — CHANGES_REQUESTED** [review]

File: ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py (hunk 2 above)
Comment (glasalm) [reply ivy_tests/test_ivy/test_frontends/test_jax/test_jax_numpy_math.py:1496]: Hello @ZoeCD! As I understand floating point data types are always signed (can hold positive and negative values) and thus it should not be a problem. If I try to use "numeric" type instead it shows error that unsigned integer types are used and therefore I think the "float" option is the best in this case. Can you verify if this is the case?

**Comment by glasalm** [comment]
@ZoeCD I have fixed your suggestions. Tests are passing again locally, but I cannot run workflow on Github actions here. Can you please run it and check if everything is correct? The test is passing locally for me. Thanks!

**Comment by ZoeCD** [comment]
The test is passing now! 👍

**Review by ZoeCD — APPROVED** [review]
